packages/documentation-site/patternfly-docs/content/design-guidelines/patterns/actions/actions.md

```diff
@@ -0,0 +1,88 @@
+1. [**Action icon (button):**](/components/button) Icon buttons are commonly used to trigger recognizable and recurring actions. For example, ellipsis icons are used to open menus.
+1. [**Action menu item:**](/components/menus/menu/design-guidelines#action-menu) To save space, actions can be placed within a menu. This is also useful when there are multiple actions that a user can take. 
+
+## Usage
+
+Use actions to make it clear and convenient for users to complete tasks in a UI. When applied intentionally, actions can make the UI more efficient and effective. 
+
+When labeling or referring to actions in your UI, make sure that you adhere to our [terminology guidelines](/ux-writing/terminology) and align any icons you use with the usage behavior outlined in our [icon design foundations.](/design-foundations/icons#all-icons) 
+
+### Deletion 
+When users can delete data objects, items, or similar resources via the UI, follow these practices to ensure that they understand the purpose and consequences of deleting something. 
+
+Deleting is different from removing. 
+- **“Delete”** refers to the destruction of information (for example, deleting a group, system, or a role.) The trash icon symbolizes deletion.
+
+- **“Remove”** refers to removing the content/system from a particular view, without the destruction of information (for example, removing something from a group or removing a user.) The solid minus-circle icon symbolizes removal.
+
+Deletion is typically initiated via buttons or menu items. When using a button trigger, apply secondary button styling. When using a menu item trigger (like within an options/kebab menu), apply default text styling. You should not use red danger styling for deletion buttons or menu items because they will both launch confirmation modals before completing the deletion. Guidance for these modals is included in the following sections. 
+#### Irreversible deletion 
+
+When resource deletion is irreversible, there is a high risk that users will lose highly important information or data. It is necessary to ensure that users understand the impact of this action. Whether a resource is critical or non-critical, you should follow the same pattern.
+
+To communicate that deletion will be permanent, use a confirmation dialog with the following features:
+
+![Confirmation modal with danger styling.](./img/critical-deletion-modal.png)
+
+1. A **warning icon** beside the title.
+1. A **labeled text input** to ask users to confirm the resource name. 
+    - An error will be shown if the user does not enter a resource name, or if the resource name does not match the resource being deleted.
+1. A **danger button** that submits the modal.
+
+Once a resource has been successfully deleted, show an [inline success alert:](/components/alert#inline-alerts-variants)
+
+![Inline success alert for deletion.](./img/critical-deletion-success.png)
+
+#### Reversible deletion 
```

Comment:
I wonder if instead of saying "reversible deletion" it would make more sense to say "removal" to align with the way you lay it out above. Deletion = do a modal, and have the red button and maybe a check box if it's really impactful deletion. But removal is more chill, may not require modal, doesn't need a red button etc